ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/firmware-bios/provision/cucushift-installer-rehearse-vsphere-upi-firmware-bios-provision-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-upi-firmware-bios-provision
```

Comment:
compared with chain `cucushift-installer-rehearse-vsphere-upi-provision`, seems no difference.

And I see that you define ENV `FIRMWARE_SET` in workflow, how about to move ENV on job config level? then we don't need to add new workflow/provision chains